### PR TITLE
Proposal of Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning

### PR DESCRIPTION
**Pull request by hsj576**
**What type of PR is this?**
This PR is a proposal of Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.

**What this PR does / why we need it**:
Based on real-world datasets provided by industry members of KubeEdge SIG AI, this project aims to build a lifelong learning benchmarking on KubeEdge-Ianvs. Namely, it aims to help all Edge AI application developers to validate and select the best-matched algorithm of lifelong learning.

### CODE REVIEW

File: docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+### 1.2 Goals
+
+Based on real-world datasets provided by industry members of KubeEdge SIG AI, this project aims to build a lifelong learning benchmarking on KubeEdge-Ianvs. Namely, it aims to help all Edge AI application developers to validate and select the best-matched algorithm of lifelong learning. It includes:
+
+1. Release new datasets for lifelong learning in KubeEdge-Ianvs.
+2. Implement BWT, FWT and other system metrics for lifelong learning in KubeEdge-Ianvs.
+
+## 2 Proposal
+
+This project will provide new real-world data sets for KubeEdge-Ianvs's lifelong learning algorithm, including a public semantic segmentation data set and a data set provided by Huawei. Each dataset released at the same time will have a supporting website to introduce the dataset. In addition, the project will further enrich the metrics method of KubeEdge-Ianvs lifelong learning algorithm, including but not limited to BWT, FWT and other methods. Data sets and metrics methods are described below.
```
Comment (luosiqi) [inline docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md:22]: The public datasets, i.e., cityscapes and synthia, are not new. The description should be corrected.

Comment (hsj576) [reply docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md]: Ok, I will correct it.

**Comment by MooreZheng** [comment]
Good job. The overall proposal makes a great test environment for lifelong learning users on Autopilot. 

As mentioned on the SIG AI meetings, the travel mode has been modified as city mode, which takes different vehicles of different cities as mobile nodes, which looks good to me.

File: docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+### 2.3 New test examples of semantic segmentation for lifelong learning
+
+#### 2.3.1 Why we need it?
+
+The way the dataset is divided during each round of training in lifelong learning tasks has a significant impact on the final model training results. If only a random partitioning method is used, there may be situations that are inconsistent with reality, such as later data appearing in earlier training sets. At the same time, changes in data distribution in different splitting methods can make it difficult to compare test results. Therefore, we urgently need a reasonable dataset splitting method as a benchmark for testing lifelong learning algorithms on semantic segmentation datasets. 
+
+Currently, we have implemented the "City Mode" test benchmark, and we will implement more various test benchmarks in the future.
+
+#### 2.3.2 An example of "City Mode" test benchmark
```
Comment (MooreZheng) [inline docs/proposals/algorithms/lifelong-learning/Cloud-Robotic AI Benchmarking for Edge-cloud Collaborative Lifelong Learning.md:80]: As mentioned on the SIG AI routine meeting, the travel mode has been modified as city mode, which takes different vehicles of different cities as mobile nodes. The current version looks good to me.

**Pull request merged**